spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -713,6 +713,11 @@ public void setConsumersPerQueue(Integer consumersPerQueue) {
 		 */
 		private String routingKey = "";
 
+		/**
+		 * The default queue name that will be used for synchronous receives.
```

Comment:
Configuration keys do not start with `The` or `A`. Please inspire yourself for the appendix.